core/src/main/java/org/apache/calcite/sql/dialect/StarRocksSqlDialect.java

```diff
@@ -109,6 +109,20 @@ public StarRocksSqlDialect(Context context) {
 
   @Override public @Nullable SqlNode getCastSpec(RelDataType type) {
     switch (type.getSqlTypeName()) {
+    case INTEGER:
+      return new SqlDataTypeSpec(
+          new SqlAlienSystemTypeNameSpec(
+              "INT",
+              type.getSqlTypeName(),
+              SqlParserPos.ZERO),
+          SqlParserPos.ZERO);
+    case BIGINT:
```

Comment:
If we don't add this case, what's the generate SQL ?

Reply:
Yes, I get it. This dialect extends MySQL dialect. That's why I generated the signed type.